internal/metrics/collectors/processes.go

```diff
@@ -56,8 +58,11 @@ func getWorkerProcesses() (int, int, error) {
 			continue
 		}
 
-		cmdlineFile := fmt.Sprintf("/proc/%v/cmdline", folder.Name())
-		content, err := ioutil.ReadFile(cmdlineFile)
+		cmdlineFile := filepath.Clean(fmt.Sprintf("/proc/%v/cmdline", folder.Name()))
```

Comment:
Hi @Juneezee 

could you possibly clarify why it is necessary to use  `filepath.Clean` ?

Thanks

Reply:
@pleshakov Hi, thanks for reviewing this PR.

`filepath.Clean` is necessary here to address the linting error, see this comment https://github.com/nginxinc/kubernetes-ingress/pull/2346#discussion_r783246802